app/src/main/java/com/example/sharewhatyoucanproject/LoginActivity.kt

```diff
@@ -32,24 +32,18 @@ class LoginActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_login)
-        supportActionBar?.hide()
-
-
         sharedPreferences = getSharedPreferences("MySharedPref", MODE_PRIVATE)
         myEdit = sharedPreferences.edit()
-        type = getIntent().getIntExtra("type", 0)
+        type = intent.getIntExtra("type", 0)
         pd = ProgressDialog(this)
         pd.setTitle("Please Wait")
         mytv = findViewById(R.id.mytv)
-
         auth = FirebaseAuth.getInstance()
         db = FirebaseFirestore.getInstance()
-
-
         if (type == 1) {
-            mytv.setText("Doner")
+            mytv.text = "Doner"
         } else if (type == 2) {
-            mytv.setText("Receiver")
+            mytv.text = "Receiver"
         }
         namearray = ArrayList()
         namearray.add("Tom")
```

Comment:
I know it's not this PR, but still, either use camelCase (so then `nameArray` instead of `name array`) or, better yet, just call it `names`.